requirements/test.txt

```diff
@@ -1,14 +1,14 @@
 -r base.txt
 -r test-acceptance.txt
 
-coverage==4.0.2
-django-nose==1.4.1
-freezegun==0.1.11
-mock==1.0.1
-moto==0.3.1
-pep8==1.7.0
-factory_boy==2.6.1
+coverage>=4.0.2,<5.0.0
+django-nose>=1.4.1,<2.0.0
+edx-i18n-tools>=0.3.4,<1.0.0
+edx-lint>=0.5.1,<1.0.0
+freezegun>=0.1.11,1.0.0
+mock>=1.0.1,<2.0.0
+moto>=0.3.1,<1.0.0
+pep8>=1.7.0,<2.0.0
+factory_boy>=2.6.1,3.0.0
 
 git+https://github.com/edx/django-pyfs.git@1.0.3#egg=django-pyfs==1.0.3
-git+https://github.com/edx/i18n-tools.git@56f048af9b6868613c14aeae760548834c495011#egg=i18n_tools
-git+https://github.com/edx/edx-lint.git@v0.4.1#egg=edx_lint==0.4.1
```

Comment:
Why were these 2 removed? I know edx/i18n-tools is needed for translation updates at least. It probably should be updated to a more recent version like [v0.3.4](https://github.com/edx/i18n-tools/releases/tag/v0.3.4), and may not be in the right location, but removing it entirely seems like a problem. edx-lint I'm less concerned about.